tests/unit/specs/scripts/url.spec.js

```diff
@@ -4,13 +4,14 @@ describe(`URL functions`, () => {
   it(`gets query params`, () => {
     const windowMock = {
       location: {
-        search: `?stargate=x&rpc=y`
+        search: `?stargate=x&network=y&graphql=z`
```

Comment:
do we still need stargate?

Reply:
yes it's still used in `cosmos-api` to simulate and broadcast